README.md

```diff
@@ -52,6 +52,16 @@ Rust bindings like those generated here, see
   - used while writing tests, to persuade the browser to select text in the
   same way as if it had been done manually.
 
-## License
+## Copyright & License
 
-[AGPL-3.0](https://www.gnu.org/licenses/agpl-3.0.txt)
+Copyright (c) 2022-2024 The Matrix.org Foundation C.I.C.
+
+Copyright (c) 2024-2025 New Vector Ltd
+
+This software is multi licensed by New Vector Ltd (Element). It can be used either:
+
+(1) for free under the terms of the GNU Affero General Public License (as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version); OR
+
+(2) for free under the terms of the GNU General Public License (as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version); OR
```

Comment:
This is the GPL licence, which does not apply to this repo(but does to element-web).

I thin this can be removed and 3) below changed to 2).